addons/netfox/rollback/rollback-synchronizer.gd

```diff
@@ -150,21 +150,35 @@ func _process_tick(tick: int):
 			NetworkRollback.process_rollback(node, NetworkTime.ticktime, tick, is_fresh)
 			_freshness_store.notify_processed(node, tick)
 
+
+
 func _record_tick(tick: int):
 	# Broadcast state we own
 	if not _auth_state_props.is_empty():
-		var broadcast = {}
-
+		var state_to_broadcast = {}
+		
+		#if (multiplayer.get_unique_id() > 1):
+			#print("Record tick state sending, peer id is: %s" % multiplayer.get_unique_id())
```

Comment:
This is always 1 (uncommented, it never prints)
Could the logic flow be explained to me? Don't clients send states directly to other clients?